packages/transaction-controller/src/constants.ts

```diff
@@ -3,9 +3,14 @@ import { TransactionType } from './types';
 export const CHAIN_IDS = {
   MAINNET: '0x1',
   GOERLI: '0x5',
+  BASE: '0x2105',
```

Comment:
Do we need to update these chain IDs?

Reply:
After using local `NETWORKS_CHAIN_ID` in mobile, it's not necessary anymore, removed in https://github.com/MetaMask/core/pull/4201/commits/71ce7e86f6dba3dc64f5687dca83737f1b4829a4